warplib/src/warp_memory.cairo

```diff
@@ -28,10 +29,14 @@ impl WarpMemoryImpl of MemoryTrait {
 
     fn insert(ref self: WarpMemory, position: felt252, value: felt252) {
         self.memory.insert(position, value);
-        self.pointer += 1;
```

Comment:
The change breaks the logic in arrayLiteral.ts. Please consider the call to insert there

Reply:
Not anymore, now we just should use `create` for static arrays